edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesPodBuilder.cs

```diff
@@ -68,10 +68,12 @@ public V1PodTemplateSpec GetPodFromModule(Dictionary<string, string> labels, IMo
             // Image
             string moduleImage = module.Config.Image;
 
+            var name = KubeUtils.SanitizeDNSValue(moduleIdentity.ModuleId);
```

Comment:
Why is this necessary? KubeUtils is a simple extension method - no need for it's own var.

Reply:
why sanitize data multiple time instead of one time and reuse multiple times?

Reply:
I see. You added "name" to V1ObjectMeta. At leat name it better...